### PR TITLE
Cap steering values sent to servo

### PR DESCRIPTION
**Pull request by VivaanBahl**
so that we don't just blindly follow what high level sends.

Note this has not been tested
Addresses #320

### CODE REVIEW

**Comment by TrevorDecker** [comment]
Has this been tested? @ramensandwich can you review it?

File: real_time/arduino_src/radio_buggy_mega/main.cpp [hunk 1]
```diff
@@ -392,6 +394,15 @@ int main(void)
 
         if(g_is_autonomous)
         {
+
+            // make sure that high-level can't send us obscenely large angles
+            if(auto_steering_angle < STEERING_LIMIT_LEFT) {
```
Comment (ramensandwich) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: Please put braces on their own lines :)

Comment (TrevorDecker) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: also why only do this if in autonomous?

Comment (ramensandwich) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: Because we're only capping the values received from auton according to the PR title.

Comment (TrevorDecker) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: ok but why not do it for teleop and auton?  I just don't see why the validation code for what an allowed angle is should be any different?

Comment (ramensandwich) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: I agree. We should be limiting both inputs

Comment (TrevorDecker) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: @ramensandwich can you patch and test this so that it limits both inputs.

**Comment by ramensandwich** [comment]
Once this has been tested it should be good to go.

**Comment by VivaanBahl** [comment]
Sean and I tested this on the mega, it properly capped the output from SweepNode

**Comment by VivaanBahl** [comment]
I sign off on this, will merge as soon as travis builds

**Pull request merged**